### ISSUE TITLE
NVDA narrates dialog title and button twice

### ISSUE THREAD

**Issue by kolaps33** [nvaccess/nvda#10003]
### Comment
The issue is reproducible with:
Chrome 75 + NVDA
Firefox 68.0.1 + NVDA
The simple example is using aria recommendations:
https://www.w3.org/TR/wai-aria-1.1/#dialog

### Steps to reproduce:
Open example: https://jsfiddle.net/kolaps33/f3nw48a6/2/
1. Navigate to "Open modal" button. 
2. Press Enter key.

### Actual behavior:
Do you like to change the settings?  dialog  content text
Got It
button
Do you like to change the settings?
button
Got It
![NVDA-narration](https://user-images.githubusercontent.com/22620150/62201407-a4e5ef00-b387-11e9-9dcd-acdd94f19980.PNG)

### Expected behavior:
NVDA narrate dialog title and focused button only once. Like:
Do you like to change the settings?  dialog  content text
Got It
button

#### NVDA installed/portable/running from source:
installed version 2019.1.1

#### NVDA version:
version 2019.1.1

#### Does the issue still occur after restarting your PC?
yes

**Comment by Adriani90** [comment]
Duplicate of #8971.

**Comment by bswanwick** [comment]
I'm seeing this same issue when using NVDA with React and Material UI.  The <Dialog> components on their demo page show this behavior.  https://material-ui.com/components/dialogs/

**Comment by Elo07** [comment]
Please where's the fix for this
